### PR TITLE
Fix for sumo 1.0.1

### LINKED ISSUE #10: Incompatible with SUMO 1.0.x
SUMO-Web3D works with SUMO 0.32 but not with the 1.0.x. series.

The culprit is probably in this section of TraCI changes:
http://sumo.dlr.de/wiki/ChangeLog#Other_3

cc @uwmyuan @katzdave

### PR DESCRIPTION
**Pull request by eugenevinitsky**
Sumo 1.0.1 made some minor changes to how subscriptions are called; this PR fixes these to make sumo web3d compatible with Sumo 1.0.1

Fix #10 

Additionally, there are some minor changes to make server.py more in accordance with Pep8 but I can remove these if desired.

### CODE REVIEW

File: sumo_web3d/server/server.py [hunk 1]
```diff
@@ -350,36 +356,49 @@ def start_sumo_executable(gui, sumo_args, sumocfg_file):
 def simulate_next_step():
     global last_lights, last_vehicles
     start_secs = time.time()
+    # Vehicles
```
Comment (danvk) [inline sumo_web3d/server/server.py]: Why move this? Admittedly this is a cryptic comment, could be something like "Update Vehicles" and "Update Lights."

File: sumo_web3d/server/server.py [hunk 2]
```diff
@@ -350,36 +356,49 @@ def start_sumo_executable(gui, sumo_args, sumocfg_file):
...
     for veh_id in traci.simulation.getDepartedIDList():
-        # SUMO will not resubscribe to vehicles that are already subscribed, so this is safe.
+        # SUMO will not resubscribe to vehicles that
+        # are already subscribed, so this is safe.
         traci.vehicle.subscribe(veh_id, TRACI_VEHICLE_CONSTANTS)
-    vehicle_response = traci.vehicle.getSubscriptionResults().items()
-    # Vehicles are automatically unsubscribed upon arrival and deleted from vehicle list on next
+
+    # acquire the relevant vehicle information
+    ids = tuple(np.unique(traci.vehicle.getIDList() +
```
Comment (danvk) [inline sumo_web3d/server/server.py]: I'd prefer not to introduce a dependence on numpy just for this.

You can make the IDs unique by putting them in a `set`.

File: sumo_web3d/server/server.py [hunk 3]
```diff
@@ -350,36 +356,49 @@ def start_sumo_executable(gui, sumo_args, sumocfg_file):
...
     # but for now we'll combine them into a single object
     vehicles.update(persons)
     vehicle_counts = Counter(v['vClass'] for veh_id, v in vehicles.items())
     round_vehicles(vehicles)
     vehicles_update = diff_dicts(last_vehicles, vehicles)
 
     # Lights
-    light_response = traci.trafficlights.getSubscriptionResults().items()
-    lights = {light_id: light_to_dict(light) for light_id, light in light_response}
+    l_ids = traci.trafficlight.getIDList()
```
Comment (danvk) [inline sumo_web3d/server/server.py]: Spell out `light_ids`, also `person_ids` above.

**Comment by eugenevinitsky** [comment]
Fixes are pushed up, thanks for the code review!

**Pull request merged**